### PR TITLE
Style and minor layout changes to sync collections with design comps

### PR DESCRIPTION
**Pull request by bmallred**
Issue #283 and [design comp](https://cloud.githubusercontent.com/assets/19935974/22039748/78acb154-dcce-11e6-982c-e15dc9bfb19d.jpg)

The collections were originally styled to an older comp and this was meant to get them closer aligned.

### CODE REVIEW

**Comment by bmallred** [comment]
@Jay-Neighbours-I72 can you take a look at this before we get it merged in to `master`

**Comment by Jay-Neighbours-I72** [comment]
@bmallred This looks good overall. Can we keep the "I don't know" button in line with the number input?

**Comment by bmallred** [comment]
@Jay-Neighbours-I72 The answer is yes, but we should probably do that in another PR because it's related to something other than the collection

File: src/components/Form/Collection/Collection.scss [hunk 1]
```diff
@@ -24,6 +24,37 @@
...
+                hr {
+                    margin-bottom: auto;
+                }
+            }
+        }
+
+        &.open {
+            .divider {
+                display: none;
+                visible: hidden;
```
Comment (floresj) [inline src/components/Form/Collection/Collection.scss]: I believe this fellow should be `visibility`

Comment (bmallred) [inline src/components/Form/Collection/Collection.scss]: Ah, good catch

**Pull request merged**